plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -136,10 +140,9 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings, U
 	isEnabled = initResult == 0 ? true : false;
 
 #if PLATFORM_WINDOWS && ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION >= 2
-	if(settings->EnableAutoCrashCapturing)
-	{
-		FPlatformMisc::SetCrashHandlingType(ECrashHandlingType::Disabled);
-	}
+	FPlatformMisc::SetCrashHandlingType(settings->EnableAutoCrashCapturing
+		? ECrashHandlingType::Disabled
+		: ECrashHandlingType::Default);
```

Comment:
This is a fallback to still keep the built-in crash reporter functional?

Reply:
Yes, without it we won't be able to restore normal crash reporter behavior within the same editor session even after disabling `EnableAutoCrashCapturing` in plugin's settings.